plugins/MultiNull/src/Logging.cpp

```diff
@@ -0,0 +1,49 @@
+
```

Comment:
The object definition: PluginLog common/include/PS2Eext.h

Example: search USBLog in plugins/USBnull/USB.cpp

Note there is also a PluginConf object in  common/include/PS2Eext.h for configuration stuff